src/modules/dashboard/components/FourOFour/FourOFour.tsx

```diff
@@ -3,7 +3,7 @@ import { defineMessages, FormattedMessage } from 'react-intl';
 import { Link } from 'react-router-dom';
 
 import Logo from '../../../../img/logo.svg';
-import NakedMole from '../../../../img/naked-mole.svg';
+import NakedMole from '../../../../img/naked-mole-404.svg';
```

Comment:
😆

Reply:
Heh

Reply:
Originally I think it was called "naked mole rat"